service/gcs/core/mockcore/mockcore.go

```diff
@@ -186,3 +161,19 @@ func (c *MockCore) ResizeConsole(pid int, height, width uint16) error {
 
 	return nil
 }
+
+// WaitContainer captures its arguments and returns a nil error.
+func (c *MockCore) WaitContainer(id string) (int, error) {
+	c.LastWaitContainer = WaitContainerCall{
+		ID = id,
+	}
+	return -1, nil
```

Comment:
note: the return value will have to be changed to use `behaviorResult` after my bridge testing change is merged in.